### PR TITLE
Most Read: fix variant example in storybook 

### LINKED ISSUE #2832: Most Read Storybook doesn't work with service variants
**Describe the bug**
https://bbc.github.io/psammead/?path=/story/components-mostread--default-rtl
When trying to use a service with a variant, it does not work.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://bbc.github.io/psammead/?path=/story/components-mostread--default-rtl'
2. Click on 'click on a service with a variant'
3. See error

**Expected behaviour**
It should show the component with the translations for that selected service.

**Screenshots**
image

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2832

**Overall change:** _Make storybook for most read work with variant service_

**Code changes:**

- _Our current test helper for most read is implemented to get text-variants by service, however when it clicks on a variant service, it would return `serbian` for example but the key value of it is `serbianLat`/`serbianCyr` in the `TEXT_VARIANTS` object. To fix this I retrieve the `variant` prop through serviceKnob and combine it with the `service` prop._
- ukchina is camel cased in `TEXT_VARIANTS` so I added an if statement to fix this.
- Non variant, services return the value `default` in the serviceKnob, so I handle this by overwriting to an empty string.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by thekp** [comment]
image
Example of it working for serbian

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.5.1-alpha.3 | [PR#2933](https://github.com/bbc/psammead/pull/2933) Fix storybook example for service with variants |
```
Comment (thekp) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 0.5.1-alpha.3 | [PR#2939](https://github.com/bbc/psammead/pull/2939) Fix storybook example for service with variants |
```

**Review by hotinglok — APPROVED** [review]
Tested each story with service knob, nice catch for ukChina. 
LGTM

File: packages/components/psammead-most-read/src/testHelpers/index.jsx [hunk 2]
```diff
@@ -18,6 +18,18 @@ const lastUpdated = ({ script, service }) => (
...
 );
 
+export const getServiceVariant = ({ service, variant = '' }) => {
+  // non variant services have a default value
+  const variantOverride =
+    variant === 'default'
+      ? ''
+      : variant.charAt(0).toUpperCase() + variant.substring(1);
+  // ukchina is camel cased in TEXT_VARIANTS
+  const serviceOverride = service === 'ukchina' ? 'ukChina' : service;
```
Comment (thekp) [inline packages/components/psammead-most-read/src/testHelpers/index.jsx]: i'm going update storybook-helper to remove the camel casing for ukchina

Comment (thekp) [reply packages/components/psammead-most-read/src/testHelpers/index.jsx]: https://github.com/bbc/psammead/pull/2940
this has been merged now, the root directory will just need a bump for it to work on live storybook

File: packages/components/psammead-most-read/src/testHelpers/index.jsx [hunk 3]
```diff
@@ -18,6 +18,16 @@ const lastUpdated = ({ script, service }) => (
...
+export const getServiceVariant = ({ service, variant = '' }) => {
+  // non variant services have a default value
+  const variantOverride =
+    variant === 'default'
+      ? ''
+      : variant.charAt(0).toUpperCase() + variant.substring(1);
+
+  return service + variantOverride;
+};
+
```
Comment (DenisHdz) [inline packages/components/psammead-most-read/src/testHelpers/index.jsx:30]: I think this looks cleaner:
```
if (variant !== 'default') {
  const variantOverride = variant.charAt(0).toUpperCase() + variant.substring(1);
  return service + variantOverride;
}

return service;
```

**Review by Bopchy — APPROVED** [review]
Looks good 👍

File: packages/components/psammead-most-read/src/testHelpers/index.jsx (hunk 3 above)
Comment (hotinglok) [reply packages/components/psammead-most-read/src/testHelpers/index.jsx:30]: Agree with this, adding this now in Khoa's absence.

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Pull request merged**